src/main/java/org/spongepowered/api/event/cause/EventContextKeys.java

```diff
@@ -22,32 +22,27 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package org.spongepowered.api.event.cause.entity.spawn;
-
-import org.spongepowered.api.Sponge;
-import org.spongepowered.api.entity.Entity;
-import org.spongepowered.api.world.weather.Weather;
+package org.spongepowered.api.event.cause;
```

Comment:
I may have said this back when we were discussing starting this PR but I'll say it again:

I feel that this class needs to be a registry of sorts. Perhaps even cataloged? Would remove some of the "rawness" of string keys and let them be looked up via the catalog framework.

Thoughts?